comp/README.md

```diff
@@ -454,6 +434,18 @@ Package trace implements the "trace" bundle, providing components for the Trace
 Package config implements a component to handle trace-agent configuration.  This
 component temporarily wraps pkg/trace/config.
 
+### [comp/trace/etwtracer](https://pkg.go.dev/github.com/DataDog/datadog-agent/comp/trace/etwtracer)
+
+*Datadog Team*: windows-agent
+
+Package apmetwtracer provides ETW events to the .Net tracer
+
 ### [comp/trace/status](https://pkg.go.dev/github.com/DataDog/datadog-agent/comp/trace/status)
 
 Package status implements the core status component information provider interface
+
+### [comp/etw](https://pkg.go.dev/github.com/DataDog/datadog-agent/comp/etw)
```

Comment:
It is alphabetical within bundles. In your case since you are a component without a bundle I opted for the simplest solution which is render the non bundle components alphabetically.

A bigger refactor could be done in a separate PR to change the tasks